docker/test/integration/runner/requirements.txt

```
@@ -104,3 +104,4 @@ zipp==1.0.0
 pyiceberg==0.7.1
 jinja2==3.1.3
 python-snappy==0.7.3
+psycopg==3.2.4
```

test is failing:
https://s3.amazonaws.com/clickhouse-test-reports/PRs/75035/b4ca55ec367b24046ecc022c98c024c449ac4830//integration_tests_aarch64_1_4/job.log
I think it should be like this:
```suggestion
psycopg[binary]==3.2.4
```